hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCreateTransitionLogic.java

```diff
@@ -116,13 +117,21 @@ public void doStateTransitionOperation(final TransactionBody contractCreateTxn,
 			key = STANDIN_CONTRACT_ID_KEY;
 		}
 
+		if (op.hasAutoRenewAccountId()) {
+			final var autoRenewAccountId = Id.fromGrpcAccount(op.getAutoRenewAccountId());
+			final var autoRenewAccount = accountStore.loadAccountOrFailWith(autoRenewAccountId, INVALID_AUTORENEW_ACCOUNT);
+			validateFalse(autoRenewAccount.isSmartContract(), INVALID_AUTORENEW_ACCOUNT);
+//			validateTrue(op.hasAdminKey(), AUTORENEW_ACCOUNT_NOT_ALLOWED); // Is this needed ?
```

Comment:
I don't think we should _require_ an `adminKey`. If a user creates a contract with `autoRenewAccount` and without `adminKey`, they are accepting the risk that if the `autoRenewAccount` is deleted, they will not be able to replace it.

Reply:
Makes sense. Thanks Michael